flink-ml-lib/src/main/java/org/apache/flink/ml/common/param/HasFeaturesCol.java

```diff
@@ -27,7 +27,10 @@
 public interface HasFeaturesCol<T> extends WithParams<T> {
     Param<String> FEATURES_COL =
             new StringParam(
-                    "featuresCol", "Features column name.", "features", ParamValidators.notNull());
+                    "featuresCol",
+                    "Name of the features column name.",
```

Comment:
According to Google results (google `name of column or column name`), it seems that the original `...column name` is more widely used than `name of column...`.

So it seems simpler to use the original `Features column name`?

And the 2nd `name` seems to be redundant here.